packages/client/src/runtime/core/model/applyModel.ts

```diff
@@ -69,7 +69,7 @@ function modelMetaLayer(dmmfModelName: string): CompositeProxyLayer {
 function modelActionsLayer(client: Client, dmmfModelName: string): CompositeProxyLayer<string> {
   // we use the javascript model name for display purposes
   const jsModelName = dmmfToJSModelName(dmmfModelName)
-  const ownKeys = getOwnKeys(client, dmmfModelName)
+  const ownKeys = Object.keys(DMMF.ModelAction).concat('count')
```

Comment:
What would happen if I try to call `aggregateRaw` on `postgresql`?

Reply:
This error:

```
PrismaClientUnknownRequestError:
Invalid `prisma.user.aggregateRaw()` invocation in
/Users/serhii/projects/prisma/reproductions/workbench/index.ts:7:21

  4
  5 async function main() {
  6   // @ts-ignore
→ 7   await prisma.user.aggregateRaw(
Operation 'aggregateRaw' for model 'User' does not match any query.
```

Reply:
I don't think it's an issue, but more out of curiosity. If you think like this can/should be improved, feel free to do it :)

Reply:
Ah, right, mine was engine-side for JSON. I'll adjust GraphQL error to be more comprehensible.

Reply:
Now for graphql it is:

```
PrismaClientValidationError:
Invalid `prisma.user.aggregateRaw()` invocation in
/Users/serhii/projects/prisma/reproductions/workbench/index.ts:7:21

  4
  5 async function main() {
  6   // @ts-ignore
→ 7   await prisma.user.aggregateRaw(
Model `User` does not support `aggregateRaw` action.

```

Reply:
Nice, thanks. Could you please add a functional test for these two?